### DISCUSSION TITLE
v3: Possibility to declare Livewire in multiple namespaces

### DISCUSSION THREAD

**Discussion by Wulfheart** [livewire/livewire#5899] [3. Feature ideas]
Given the following structure
```
- apps
  - first
    - livewire
  - second
    - livewire
```
I would like to declare in the configuration that the two livewire paths/namespaces are the root namespaces.
Is this possible?

**Comment by joshhanley** [comment]
@Wulfheart not sure if this is possible at the moment. But PRs are welcome!
